nemo_curator/metrics/start_prometheus_grafana.py

```diff
@@ -0,0 +1,102 @@
+import argparse
+import os
+import sys
+import time
+
+from loguru import logger
+
+from nemo_curator.core.utils import get_free_port
+from nemo_curator.metrics.constants import (
+    DEFAULT_GRAFANA_WEB_PORT,
+    DEFAULT_NEMO_CURATOR_METRICS_PATH,
+    DEFAULT_PROMETHEUS_WEB_PORT,
+)
+from nemo_curator.metrics.utils import (
+    download_and_extract_prometheus,
+    download_grafana,
+    is_grafana_running,
+    is_prometheus_running,
+    launch_grafana,
+    run_prometheus,
+    write_grafana_configs,
+)
+
+# ----------------------------
+# Helpers
+# ----------------------------
+
+
+def get_ray_client(
```

Comment:
I am confused. I thought this should be part of `RayClient`, not a separate function?